openedx/core/djangoapps/theming/models.py

```diff
@@ -0,0 +1,39 @@
+"""
+Django models supporting the Comprehensive Theming subsystem
+"""
+from django.db import models
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+
+class SiteTheme(models.Model):
```

Comment:
As with Otto we want both a SiteTheme model and a SiteConfiguration model, however the SiteConfiguration model is not required for this particular changeset -- we will introduce it with a follow-on PR that tackles WL-343.